extensions/amp-ima-video/0.1/amp-ima-video.js

```diff
@@ -71,17 +74,23 @@ class AmpImaVideo extends AMP.BaseElement {
         'The data-tag attribute is required for <amp-video-ima> and must be ' +
             'https');
 
-    // Set data-sources attribute based on source child elements.
     const sourceElements = this.element.getElementsByTagName('source');
-    if (sourceElements.length > 0) {
-      const sources = [];
-      toArray(sourceElements).forEach(source => {
-        if (!this.preconnectSource_) {
-          this.preconnectSource_ = source.src;
+    const trackElements = this.element.getElementsByTagName('track');
+    const childElements =
+        toArray(sourceElements).concat(toArray(trackElements));
+    if (childElements.length > 0) {
+      const children = [];
+      childElements.forEach(child => {
+        // Save the first source and first track to preconnect.
+        if (child.tagName == 'SOURCE' && !this.preconnectSource_) {
+          this.preconnectSource_ = child.src;
+        } else if (child.tagName == 'TRACK' && !this.preconnectTrack_) {
```

Comment:
could you please update the `.md` file to reflect that `track` and `source` children are supported?